cmd/infracost/comment.go

```diff
@@ -35,7 +36,7 @@ func commentCmd(ctx *config.RunContext) *cobra.Command {
 	return cmd
 }
 
-func buildCommentBody(ctx *config.RunContext, paths []string, mdOpts output.MarkdownOptions) ([]byte, error) {
+func buildCommentBody(cmd *cobra.Command, ctx *config.RunContext, paths []string, mdOpts output.MarkdownOptions) ([]byte, error) {
```

Comment:
I tried different approaches but this makes the most sense as it's part of the command anyway:
- Introduce "commentOptions" with `DryRun` attribute and pass it here, but the self-hosted warning would be tricky to print.
- Sending the run in each github/gitlab/azure comment functions seemed too repetitive.

Reply:
My only thought is should dryRun be part of the the RunContext, but we're not using it elsewhere so it seems ok to me like this.